awscli/data/codebuild/2016-10-06/completions-1.sdk-extras.json

```diff
@@ -0,0 +1,102 @@
+{
+  "merge": {
+    "resources": {
+      "Projects": {
+        "operation": "ListProjects",
+        "resourceIdentifier": {
+          "project": "projects[]"
+        }
+      }
+    },
+    "operations": {
+      "BatchDeleteBuilds": {
```

Comment:
If we are going to touch builds, it probably be helpful to add `BatchGetBuilds` and `RetryBuild` as well.

Reply:
`RetryBuild` is in script generated completions

Reply:
I thought it was only `StopBuild` that in the completion file: https://github.com/aws/aws-cli/blob/2cb3e286ab4b1abbf0496f5c80cc0dc47e3e8dc9/awscli/data/codebuild/2016-10-06/completions-1.json#L12 and not `RetryBuild`.

Reply:
In such a case we need to run `gen-server-completions` and push updated completion files, I thought we did in CI

Reply:
Gotcha. Yeah let's then just run the `gen-server-completions` script to update this completions model for `codebuild` and then fill the missing using the extra's model. In the long run, it would be nice to see the `gen-server-completions` script catch the project related completions.